docs/core-study-details.md

```diff
@@ -27,6 +27,25 @@ for more information.
 
 # Table format
 
+Unless otherwise noted, column names correspond to a FHIR path relative to the resource
+table in question, with resources delineated by underscores. So, 'category_code' in the 
+Condition table would correspond to the code element inside of the category element of
+the Condition resource.
+
+Common exceptions to this rule:
```

Comment:
Other things you *could* mention, but you don't need to, just thoughts:
- DateTime and Period fields are usually chopped up.
- Often References are represented with a toplevel `xxx_ref` field, and sometimes that field lives alongside a simple ID version of same.